app/assets/stylesheets/petitions/views/_sign.scss

```diff
@@ -6,3 +6,7 @@
   padding-left: $gutter * 1.5;
   margin-bottom: $gutter * 1.3;
 }
+
+#signature_postcode, #petition_creator_signature_postcode {
+  text-transform: uppercase;
+}
```

Comment:
We do have server-side validation. We the build team know this isn't case sensitive but plenty of users won't assume that. This is just a nicety for the user (I think it only affects how the `input` content is displayed).